test/ipc-ut.cc

```diff
@@ -42,7 +40,7 @@ TEST(IPCTest, Positive) {
     EXPECT_FALSE(ec);
     close(sockets[kChildIdx]);
     close(fileFd);
-    return;
+    exit(0);
```

Comment:
I fear that I will have a hard time remembering this.  `return()` actually gives control back up to the unit test handler, so `fork()` will resume.  My brain wants to think of each unit test as an isolated `main()`.

Reply:
(there's nothing to do about that, unless you can propose a patch for my brain)